lib/api/1.1/southbound/router.js

```diff
@@ -0,0 +1,37 @@
+// Copyright 2015, EMC, Inc.
```

Comment:
Yes, good suggestion.
Unfortunately, v2 API will adopt `swagger yaml` to replace routers in express style. So, I think we could save this effort